tests/all_tests.py

```diff
@@ -118,14 +118,14 @@ def test(suppress_unit_tests, suppress_javascript_unit_tests, unittest_glob):
     if not suppress_unit_tests:
         print("\nUnit test result summary\n------------------------\n")
         if bad_tests:
-            print(("\nFailed the following unit tests.\n%s"
-                   % '\n'.join(bad_tests)))
+            print("\nFailed the following unit tests.\n%s"
```

Comment:
Should it be better to say: "The following unit tests failed:" or "Failing unit tests:"?